lightkurve/correctors.py

```diff
@@ -209,6 +216,37 @@ def get_cbv_url(self):
 
         return self.cbv_base_url + cbv_file
 
+    def plot_cbvs(self, cbvs=[1, 2], ax=None):
+        '''Plot the CBVs for a given list of CBVs
+
+        Parameters
+        ----------
+        cbvs : list of ints
+            The list of cotrending basis vectors to fit to the data. For example,
+            [1, 2] will fit the first two basis vectors.
+        ax : matplotlib.pyplot.Axes.AxesSubplot
+            Matplotlib axis object. If none, one will be generated.
+        Returns
+        -------
+        ax : matplotlib.pyplot.Axes.AxesSubplot
+            Matplotlib axis object
+        '''
+        if ax is None:
+            _, ax = plt.subplots(1)
+        cbv_array, time = self._get_cbv_data(cbvs)
+        for idx, cbv in enumerate(cbv_array):
+            ax.plot(time, cbv+idx/10., label='{}'.format(idx + 1))
+        ax.set_yticks([])
+        ax.set_xlabel('Time (MJD)')
```

Comment:
Are you sure this is MJD, not BKJD?

Reply:
It says TIME_MJD in the CBV file and it has values  ~55000 so I thought it was MJD